src/main/java/com/gmail/goosius/townycultures/utils/PresetCulturesUtil.java

```diff
@@ -0,0 +1,115 @@
+package com.gmail.goosius.townycultures.utils;
+
+import com.gmail.goosius.townycultures.metadata.TownMetaDataController;
+import com.gmail.goosius.townycultures.objects.PresetCulture;
+import com.gmail.goosius.townycultures.settings.Settings;
+import com.palmergames.bukkit.towny.TownyAPI;
+import com.palmergames.bukkit.towny.object.Coord;
+import com.palmergames.bukkit.towny.object.Town;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class PresetCulturesUtil {
+
+    //Note: In this map, the keys are all lowercase
+    private static final Map<String, PresetCulture> presetCulturesMap = new HashMap<>();
+
+    public static void clearPresetCultures() {
+        presetCulturesMap.clear();
+    }
+
+    public static void loadPresetCultures() {
+        List<PresetCulture> presetCulturesList = Settings.getPresetCulturesList();
+        for(PresetCulture presetCulture: presetCulturesList)
+        {
+            presetCulturesMap.put(presetCulture.getName().toLowerCase(), presetCulture);
+        }
+    }
+
+    /**
+     * If Preset Cultures is enabled, ensure all towns have a valid preset culture.
+     */
+    public static void sanitizeTownCultures() {
+
+        if (Settings.isPresetCulturesEnabled()) {
+            {
+                for (Town town : TownyAPI.getInstance().getTowns()) {
+                    if (!TownMetaDataController.hasTownCulture(town) || !PresetCulturesUtil.isPresetCulture(TownMetaDataController.getTownCulture(town))) {

```

Comment:
`!PresetCulturesUtil.isPresetCulture(TownMetaDataController.getTownCulture(town))`

This is meant to wipe any custom culture in favour of the preset culture belonging to the area.